### ISSUE TITLE
Is the logs of the train loss with the pre-trained Tacotron2 model supplied?

### ISSUE THREAD

**Issue by chlee01102** [NVIDIA/tacotron2#368]
I use the LJspeech to re-train a new Tactotron2 model with different parameters (hparams.py) and get the training log as following:

Epoch:500 3GPU(3*1080ti, batch size: 32 ) 
FP16 Run: False
Dynamic Loss Scaling: True
Distributed Run: True
cuDNN Enabled: True
cuDNN Benchmark: False
Initializing Distributed
Done initializing distributed
Epoch: 0
Train loss 0 48.220909 Grad Norm 13.484264 5.71s/it
Validation loss 0: 37.061010  
Saving model and optimizer state at iteration 0 to /data/models/ljspeech_models/waveglow_04242020/checkpoint_0
Train loss 1 35.229748 Grad Norm 25.020286 3.79s/it
Train loss 2 16.508032 Grad Norm 17.146819 3.99s/it
Train loss 3 9.564240 Grad Norm 13.827114 3.99s/it
Train loss 4 5.843519 Grad Norm 6.745416 3.99s/it
Train loss 5 5.946629 Grad Norm 7.371412 4.03s/it
Train loss 6 6.061152 Grad Norm 7.941638 4.01s/it
Train loss 7 5.797815 Grad Norm 6.959098 3.97s/it
….
Validation loss 61000:  0.325403  
Saving model and optimizer state at iteration 61000 to /data/models/ljspeech_models/waveglow_04242020/checkpoint_61000
…
Validation loss 64000:  0.323265  
Saving model and optimizer state at iteration 64000 to /data/models/ljspeech_models/waveglow_04242020/checkpoint_64000
…
Train loss 64998 0.302735 Grad Norm 0.958112 4.36s/it
Train loss 64999 0.314525 Grad Norm 0.615372 4.37s/it

I want to compare the Train loss of my training model with the pre-trained model.
Is the logs of the train loss with the pre-trained Tacotron2 model supplied?
Thanks for help.

**Comment by rafaelvalle** [comment]
You can use the pre-trained model and compute the loss over all samples without doing updates, e.g. with torch.no_grad().
